deployments/flp-service.yml

```diff
@@ -0,0 +1,155 @@
+# Example deployment for manual testing with flp
+# It requires loki to be installed
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: netobserv-agent
+  labels:
+    k8s-app: netobserv-agent
+spec:
+  selector:
+    matchLabels:
+      k8s-app: netobserv-agent
+  template:
+    metadata:
+      labels:
+        k8s-app: netobserv-agent
+    spec:
+      serviceAccountName: netobserv-account
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+      - name: netobserv-agent
+        image: quay.io/mmaciasl/netobserv-agent:main
+        #      imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        env:
+          - name: FLOWS_TARGET_HOST
+            value: "flp"
+          - name: FLOWS_TARGET_PORT
+            value: "9999"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: flp
+  labels:
+    k8s-app: flp
+spec:
+  ports:
+    - port: 9999
+      protocol: TCP
+      targetPort: 9999
+      name: flp
+  selector:
+    k8s-app: flp
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flp
+  labels:
+    k8s-app: flp
+spec:
+  selector:
+    matchLabels:
+      k8s-app: flp
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: flp
+    spec:
+      automountServiceAccountToken: true
+      serviceAccountName: netobserv-controller-manager
+      containers:
+        - name: packet-counter
+          image: quay.io/mmaciasl/flowlogs-pipeline:latest
```

Comment:
```suggestion
          image: quay.io/netobserv/flowlogs-pipeline:latest
```

Reply:
D'oh! Didnt realize that. Let me amend them in another PR

Reply:
no worries, I know it's a pain to switch in all configs

Reply:
https://github.com/netobserv/netobserv-agent/pull/10/files